jazz_services-handler/test/test.js

```diff
@@ -14,686 +14,164 @@
+					"eventSourceARN": "arn:aws:kinesis:us-east-1:192006145812:stream/gitlab180314-events-hub-prod"
+				}
+			]
+		};
+		context = awsContext();
+		context.functionName = context.functionName+"-test"
+		tokenResponseObj = {
+			statusCode : 200,
+			body : {
+				"test": "value",
+				data: {
+					token: "abc"
+				}
+			},
+			response: "myHeader"
+		};
+		configData = config(context);
+		callback = (value) => {
+			return value;
+		};
+	});
+
+	it('getToken should reject for response  with out response.body.data', function() {
+	//	var stub = sinon.stub(rp, 'post').returns(Promise.resolve(tokenResponseObj));
+		var getTokenPromise = index.getToken(configData);
+		expect(rp(getTokenPromise).then((res) => {
+      		console.log("Success: " + JSON.stringify(res)); 
+     	return res; })).to.become(tokenResponseObj.body.data.token);
+	});
+
+	it('validateAuthToken should return success promise for response data with status code 200', function() {
+		var validate = index.validateAuthToken(tokenResponseObj);
+		expect(validate.then(function(res) {	
+		return res; })).to.become(tokenResponseObj.body.data.token);
+	});	
+
+	it('validateAuthToken should reject for response data with status code 400', function() {
+		tokenResponseObj.statusCode = 400;
+		var message = "User is not authorized to access this service";
+		var validate = index.validateAuthToken(tokenResponseObj);
+		expect(validate.then(function(res) {
+		return res; })).to.be.rejectedWith(message);
+	});
+
+	it('validateAuthToken should reject for response data with out response.body', function() {
+		tokenResponseObj.body = {};
+		var message = "User is not authorized to access this service";
+		var validate = index.validateAuthToken(tokenResponseObj);
+		expect(validate.then(function(res) {
+		return res; })).to.be.rejectedWith(message);
+	});
+
+	it('validateAuthToken should reject for response  with out response.body.data', function() {
+		tokenResponseObj.body.data = null;
+		var message = "User is not authorized to access this service";
+		var validate = index.validateAuthToken(tokenResponseObj);
+		expect(validate.then(function(res) {
+		return res; })).to.be.rejectedWith(message);
+	});
+
+	it('handleError should return error response json for valid input', function() {
+		var errorJson = {"failure_code" : 400,"failure_message" : "Unauthorized"};
+		var handleError = index.handleError(400,"Unauthorized");
+		
+		var handleErrorStub = sinon.stub(index, "handleError");
+		handleErrorStub.withArgs(400,"Unauthorized").returns(errorJson);		
+	});
+
+	it('processRecords should reject error for invalid input', function() {
+		var payload = {};
+		var authToken = tokenResponseObj.body.data.token;
+		var message = "event is not defined";
+		var processRecords = index.processRecords(configData, authToken);
+		expect(processRecords.then(function(res) {
+		return res; })).to.be.rejectedWith(message);		
+	});	
+
+	it('checkInterest should reject error for invalid input', function() {
```

Comment:
The test names for this and line 151 are exactly same. Lets give them the right description